nilearn/datasets/tests/test_atlas.py

```diff
@@ -398,3 +398,22 @@ def test_fetch_coords_dosenbach_2010():
 
     bunch = atlas.fetch_coords_dosenbach_2010(ordered_regions=False)
     assert_true(np.any(bunch.networks != np.sort(bunch.networks)))
+
+
+@with_setup(setup_mock, teardown_mock)
+@with_setup(tst.setup_tmpdata, tst.teardown_tmpdata)
+def test_fetch_atlas_allen_2011():
+    bunch = atlas.fetch_atlas_allen_2011(data_dir=tst.tmpdir, verbose=0)
+    keys = ("all_unthresh_tmaps",
```

Comment:
You missed to change here to "maps", "rsn28", "comps", I guess. Can you please have a look ?